contrib/exec-healthz/Makefile

```diff
@@ -0,0 +1,17 @@
+all: push
+
+# 0.0 shouldn't clobber any released builds
+TAG = 0.0
+PREFIX = gcr.io/google_containers/healthz-server
+
+server: simpleHealthzServer.go
+	CGO_ENABLED=0 GOOS=linux go build -a -installsuffix cgo -ldflags '-w' -o healthz_server ./simpleHealthzServer.go
```

Comment:
Of course I have no excuse for picking simpleHealthzServer other than wanting to mimic simpleHTTPServer because it's awesome